modules/auxiliary/admin/mssql/mssql_escalate_dbowner.rb

```diff
@@ -20,12 +20,24 @@ def initialize(info = {})
       'License'        => MSF_LICENSE,
       'References'     => [[ 'URL','http://technet.microsoft.com/en-us/library/ms188676(v=sql.105).aspx']]
     ))
+
+    register_options(
+      [
+        Opt::RHOST,
+        Opt::RPORT(1433),
+        OptString.new('USERNAME', [ false, 'The username to authenticate as', 'sa']),
+        OptString.new('PASSWORD', [ false, 'The password for the specified username', '']),
+        OptBool.new('TDSENCRYPTION', [ true, 'Use TLS/SSL for TDS data "Force Encryption"', false]),
+        OptBool.new('USE_WINDOWS_AUTHENT', [ true, 'Use windows authentication (requires DOMAIN option set)', false]),
+      ])
```

Comment:
Similar to the above; We'll want to still have a MSSQL client mixin, which knows how to instantiate a Client

This will allow for us to not need to copy/pasta registering these options here